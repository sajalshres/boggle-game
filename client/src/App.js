import React from 'react';
import Header from './components/Header/index';
import Main from './components/Main/index';
import Footer from './components/Footer/index';
import './App.scss';

function App() {
  return (
    <div className="App">
      <Header />
      <Main />
      <Footer />
    </div>
  );
}

export default App;
