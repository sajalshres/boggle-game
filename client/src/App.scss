$height-header: 100px;
$height-footer: 50px;

.App {
    position: relative;
    display: grid;
    grid-template-columns: 100%;
    grid-template-rows: $height-header 1fr $height-footer;
    grid-template-areas:
        'header'
        'main'
        'footer';
    width: 60%;
    min-width: 800px;
    height: 100vh;
    margin: 0 auto;
    justify-items: center;
}