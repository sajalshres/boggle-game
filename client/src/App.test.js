import React from 'react';
import ReactDom from 'react-dom';

import Enzyme, { shallow, render, mount } from 'enzyme';
import toJson from 'enzyme-to-json';
import Adapter from 'enzyme-adapter-react-16';

import App from './App';
import { faItalic } from '@fortawesome/free-solid-svg-icons';

Enzyme.configure({ adapter: new Adapter() });

it('renders correctly enzyme', () => {
  const wrapper = shallow(<App />);

  expect(toJson(wrapper)).toMatchSnapshot();
});
