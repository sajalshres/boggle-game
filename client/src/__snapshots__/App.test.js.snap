// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly enzyme 1`] = `
<Provider
  store={
    Object {
      "dispatch": [Function],
      "getState": [Function],
      "replaceReducer": [Function],
      "subscribe": [Function],
      Symbol(observable): [Function],
    }
  }
>
  <BrowserRouter>
    <div
      className="App"
    >
      <Header />
      <main
        className="main"
      >
        <Switch>
          <Route
            component={[Function]}
            exact={true}
            path="/"
          />
          <Route
            component={
              Object {
                "$$typeof": Symbol(react.memo),
                "WrappedComponent": [Function],
                "compare": null,
                "displayName": "Connect(Boogle)",
                "type": [Function],
              }
            }
            path="/game"
          />
          <Route
            component={
              Object {
                "$$typeof": Symbol(react.memo),
                "WrappedComponent": [Function],
                "compare": null,
                "displayName": "Connect(Rank)",
                "type": [Function],
              }
            }
            path="/rank"
          />
        </Switch>
      </main>
      <Footer />
    </div>
  </BrowserRouter>
</Provider>
`;
